### ISSUE TITLE
[US - DEV] As MILES HARDIN, I want to receive an email so that I can know when someone made a new comment on an idea I own

### ISSUE THREAD

**Issue by JulieMass** [dev-launchers/dev-launchers-platform#917]
### Considerations:
As an idea owner, Miles Hardin wants to receive email notifications when someone makes a new comment on his idea. This feature will help him stay updated on the activity of his ideas and respond to comments in a timely manner. 

Additionally, the email notifications aim to increase user retention on the IdeaSpace platform. 

**Important considerations:**
- The email design has been created by Anna from the Universal Design team and can be found on the following [Figma ](https://www.figma.com/file/B8QY4HfTxJVlvpCaRjyiXw/Email-Template-Design?type=design&node-id=301%3A3231&mode=design&t=VlmjdHo6WkfxutwI-1)
- We’ll be using the Sendgrid tool as an [emailing tool](https://sendgrid.com/pricing/?utm_source=google&utm_medium=cpc&utm_term=sendgrid%20pricing&utm_campaign=SendGrid_G_S_NAMER_Brand_Tier1&cq_plac=&cq_net=g&cq_pos=&cq_med=&cq_plt=gp&gclid=EAIaIQobChMI74qeh-aT_QIVvBXUAR0NMgZnEAAYASAAEgKC9PD_BwE ).  **Note**: We have a dev launchers account for Sendgrid so we'll share the data to access it. 

### Acceptance criteria:
- The mail notification follows the [template from the Universal Design team.](https://www.figma.com/file/B8QY4HfTxJVlvpCaRjyiXw/Email-Template-Design?type=design&node-id=301%3A3231&mode=design&t=VlmjdHo6WkfxutwI-1) 
- When a user comments on an idea owned by Miles, he should receive an email notification within 5 minutes.
- The email notification should be sent to the email address associated with Miles Hardin's account.
- If multiple comments are made on the same idea, Miles should receive a separate email notification for each comment.
- If Miles Hardin owns multiple ideas and comments are made on different ideas, he should receive separate email notifications for each idea.
- The email notification should be sent from our email service [Sendgrid](https://sendgrid.com/pricing/?utm_source=google&utm_medium=cpc&utm_term=sendgrid%20pricing&utm_campaign=SendGrid_G_S_NAMER_Brand_Tier1&cq_plac=&cq_net=g&cq_pos=&cq_med=&cq_plt=gp&gclid=EAIaIQobChMI74qeh-aT_QIVvBXUAR0NMgZnEAAYASAAEgKC9PD_BwE) and not marked as spam by email providers.
- The email notifications is web responsive and was tested thoroughly and works seamlessly on all devices and [target browsers ](https://docs.google.com/spreadsheets/d/1upYP1mvHkU3sRBg4gCeGG4P63tO5-whn6VKLMlr6qYI/edit?usp=sharing): Chrome, Microsoft Edge, Safari, Firefox and Opera.

----------------------------------------------------
### Sub-tasks:
- [X] UXR ([Usability test report in EnjoyHQ](https://app.enjoyhq.com/sh/pj/f765e04d-45f9-4f49-ab37-b9b48ccc27ba))
- [X] UX design (by Anna from the Universal Design Team)
- [ ] Development
- [ ] QA 

----------------------------------------------------
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdHZOIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--2a7ec028161a41e411192a134e6d3065e2157fe1/image.png)

**Comment by JulieMass** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/IdeaSpace-Team-6350185d9843157427cc900b/issues/dev-launchers/dev-launchers-platform/917?planning-poker) with Zenhub @alicefate @dbradham @jordanmcousins @JulieMass @katehirschman @Nouri-Anouar @rashikashaw @RETprojects

**Comment by dbradham** [comment]
Just to clarify the scope here-this is just 1 email design?

**Comment by JulieMass** [comment]
> Just to clarify the scope here-this is just 1 email design? (for Miles when someone comments on his idea)

Yes, that is correct. It also involves the designer to get used to our emailing tool and design an email template that can work with it. So they might need to explore the tool a bit at the beginning I guess. 

Do you think we should cover more? Let me know =)

**Comment by JulieMass** [comment]
We learned during the last organization sprint review that the Universal design team is conducting a usability test on the email templates. Therefore, we have decided to move this ticket to the icebox.

@Nouri-Anouar , please keep us updated on the progress of the usability testing for the email templates. Once the design for this email is refined with the improvements after the research and is considered approved, our team can start developing. Meanwhile, we have moved this ticket to our icebox and added a dependency on your ticket #1300. Thank you!

(CC. @dbradham )
